inspirehep/modules/records/jsonschemas/records/authors.json

```diff
@@ -105,13 +104,13 @@
                             "curated_relation": {
                                 "type": "boolean"
                             },
-                            "recid": {
-                                "type": "integer",
-                                "description": "Corresponding record ID in the institution database"
-                            },
                             "name": {
                                 "type": "string",
                                 "description": "The raw institution name"
+                            },
+                            "record": {
+                                "description": "Corresponding URI for the institution record",
```

Comment:
Can you actually put anything different than `$ref` in the object? I thought you can only put `$ref` since this will be substituted by the referred object.